test/610_proxy_wait_for_weave_test.sh

```diff
@@ -0,0 +1,33 @@
+#! /bin/bash
+
+. ./config.sh
+
+docker_proxy_on() {
+    host=$1
+    shift 1
+    greyly echo "Docker (with proxy) on $host: $@" >&2
+    docker -H tcp://$host:12375 $@
+}
+
+start_suite "Proxy waits for weave to be ready before running container commands"
+weave_on $HOST1 launch-proxy
+assert_raises "docker_proxy_on $HOST1 run -e 'WEAVE_CIDR=10.2.1.1/24' --name=c1 gliderlabs/alpine ifconfig ethwe" 0
```

Comment:
I was going to say that we favour `ip` over the deprecated `ifconfig`, but I've just checked and we are using `ifconfig` in one other test and also mention it in `dockerless.txt`. Should we standardise on `ip` @rade?